### PR TITLE
fix: add tw-text-jumbo to the safelist since it's only used in conten…

### PR DESCRIPTION
**Pull request by ryan-ludwig**
…tful

### CODE REVIEW

**Comment by ryan-ludwig** [comment]
This one had me scratching my head for a while, til I realized I couldn't find the class used in UI anywhere else.

**Comment by mcstover** [comment]
@ryan-ludwig do you think we should go ahead and add the new themeable text classes

**Comment by ryan-ludwig** [comment]
@mcstover Yeah, I think that makes sense to have all the themeable colors + all the type styles whitelisted. I'll update this PR.

**Review by mcstover — APPROVED** [review]

File: tailwind.purge.safelist.txt [hunk 1]
```diff
@@ -38,3 +39,33 @@ tw-pb-8
 md:tw-pb-8
 lg:tw-pb-8
 xl:tw-pb-8
+
+# Themable color classes
+tw-bg-primary
```
Comment (ryan-ludwig) [inline tailwind.purge.safelist.txt:44]: Hmm, now thinking we wouldn't ever set a `tw-bg-*` class within a contentful span.

**Review by mcstover — APPROVED** [review]

**Review by emuvente — APPROVED** [review]

**Pull request merged**